save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -0,0 +1,213 @@
+            setState {
+                image = avatar
+            }
+        }
+    }
+
+    @Suppress("TOO_LONG_FUNCTION", "LongMethod", "MAGIC_NUMBER")
+    override fun RBuilder.render() {
+        div("d-sm-flex align-items-center justify-content-center mb-4") {
+            h1("h3 mb-0 text-gray-800") {
+                +props.organizationName
+            }
+        }
+
+        div("row justify-content-center") {
+            // ===================== LEFT COLUMN =======================================================================
+            div("col-2 mr-3") {
+                div("text-xs text-center font-weight-bold text-primary text-uppercase mb-3") {
+                    +"Organization"
+                }
+
+                div {
+                    attrs["style"] = kotlinx.js.jso<CSSProperties> {
+                        position = "relative".unsafeCast<Position>()
+                        textAlign = "center".unsafeCast<TextAlign>()
+                    }
+                    label {
+                        input(type = InputType.file) {
+                            attrs.hidden = true
+                            attrs {
+                                onChangeFunction = { event ->
+                                    val target = event.target as HTMLInputElement
+                                    postImageUpload(target)
+                                }
+                            }
+                        }
+                        attrs["aria-label"] = "Change avatar owner"
+                        img(classes = "avatar avatar-user width-full border color-bg-default rounded-circle") {
+                            attrs.src = state.image?.path?.let {
+                                "/api/avatar$it"
+                            }
+                                ?: run {
+                                    "img/company.svg"
+                                }
+                            attrs.height = "260"
+                            attrs.width = "260"
+                        }
+                    }
+                }
+
+                div("position-relative") {
+                    attrs["style"] = kotlinext.js.jso<CSSProperties> {
+                        position = "relative".unsafeCast<Position>()
+                        textAlign = "center".unsafeCast<TextAlign>()
+                    }
+                    img(classes = "width-full color-bg-default") {
+                        attrs.src = "img/green_square.png"
+                        attrs.height = "200"
+                        attrs.width = "200"
+                    }
+                    div("position-absolute") {
+                        attrs["style"] = kotlinext.js.jso<CSSProperties> {
+                            top = "40%".unsafeCast<Top>()
+                            left = "40%".unsafeCast<Left>()
+                        }
+                        h1(" mb-0 text-gray-800") {
+                            +"4.5"
```

Comment:
Please replace with actual value, or add fixme comment and disable

Reply:
Yes. Added fixme and supported OrganizationView for TopBar